src/server/db.json

```diff
@@ -3,31 +3,31 @@
     {
       "id": 0,
       "name": "Build a fence",
-      "contactId": "0",
+      "contactId": 0,
       "start": "2018-09-01T10:00:00Z",
       "end": "2018-09-01T11:00:00Z",
       "location": "Brisbane"
     },
     {
       "id": 1,
       "name": "Build a shed",
-      "contactId": "1",
+      "contactId": 1,
       "start": "2018-09-01T10:15:00Z",
       "end": "2018-09-01T11:00:00Z",
       "location": "Brisbane"
     },
     {
       "id": 2,
       "name": "Shield some wiring",
-      "contactId": "0",
+      "contactId": 0,
       "start": "2018-09-01T09:00:00Z",
       "end": "2018-09-01T13:00:00Z",
       "location": "Brisbane"
     },
     {
       "id": 3,
       "name": "Pick up a trailer",
-      "contactId": "0",
+      "contactId": 0,
```

Comment:
We have a bug that I've been meaning to fix in here with contacts. Two of them have the same id. Be good to fix that up

Reply:
Good catch.